app/javascript/data/advancedFilterOptions.js

```diff
@@ -0,0 +1,266 @@
+  },
+  {
+    name: 'preferred-contact-time',
+    title: 'Preferred Contact Time (Select)',
+    description: 'Monitoree preferred contact time',
+    type: 'select',
+    options: ['Morning', 'Afternoon', 'Evening', '']
+  },
+  {
+    name: 'primary-language',
+    title: 'Primary Language (Select)',
+    description: 'Monitoree primary language',
+    type: 'select',
+    options: supportedLanguages.languages
+      .map(lang => {
+        return lang.name;
+      })
+      .concat(['']),
+  },
+  {
+    name: 'risk-exposure',
+    title: 'Exposure Risk Assessment (Select)',
+    description: 'Monitoree exposure risk assessment',
+    type: 'select',
+    options: ['High', 'Medium', 'Low', 'No Identified Risk', '']
+  },
+
+  /* NUMBER FILTER OPTIONS */
+  {
+    name: 'age',
+    title: 'Age (Number)',
+    description: 'Current Monitoree Age',
+    type: 'number',
+    allowRange: true
+  },
+  {
+    name: 'manual-contact-attempts',
+    title: 'Manual Contact Attempts (Number)',
+    description: 'All records with the specified number of manual contact attempts',
+    type: 'number',
+    options: ['Successful', 'Unsuccessful', 'All']
+  },
+
+  /* DATE FILTER OPTIONS */
+  {
+    name: 'enrolled',
+    title: 'Enrolled (Date)',
+    description: 'Monitorees enrolled in system during specified date range',
+    type: 'date',
+  },
+  {
+    name: 'latest-report',
+    title: 'Latest Report (Date)',
+    description: 'Monitorees with latest report during specified date range',
+    type: 'date',
+  },
+  {
+    name: 'last-date-exposure',
+    title: 'Last Date of Exposure (Date)',
+    description: 'Monitorees who have a last date of exposure during specified date range',
+    type: 'date',
+  },
+  {
+    name: 'symptom-onset',
+    title: 'Symptom Onset (Date)',
+    description: 'Monitorees who have a symptom onset date during specified date range',
+    type: 'date',
+  },
+
+  /* RELATIVE DATE FILTER OPTIONS */
+  {
+    name: 'enrolled-relative',
+    title: 'Enrolled (Relative Date)',
+    description: 'Monitorees enrolled in system during specified date range (relative to the current date)',
+    type: 'relative',
+    hasTimestamp: true,
+  },
+  {
+    name: 'latest-report-relative',
+    title: 'Latest Report (Relative Date)',
+    description: 'Monitorees with latest report during specified date range (relative to the current date)',
+    type: 'relative',
+    hasTimestamp: true,
+  },
+  {
+    name: 'last-date-exposure-relative',
+    title: 'Last Date of Exposure (Relative Date)',
+    description: 'Monitorees who have a last date of exposure during specified date range (relative to the current date)',
+    type: 'relative',
+    hasTimestamp: false,
+  },
+  {
+    name: 'symptom-onset-relative',
+    title: 'Symptom Onset (Relative Date)',
+    description: 'Monitorees who have a symptom onset date during specified date range (relative to the current date)',
+    type: 'relative',
+    hasTimestamp: false,
+  }
+]
```

Comment:
@holmesie thoughts on adding [this eslint plugin](https://eslint.org/docs/rules/eol-last) along with the other eslint plugins we were thinking of including? 

https://eslint.org/docs/rules/comma-dangle
https://eslint.org/docs/rules/semi

and maybe also: https://eslint.org/docs/rules/sort-imports

Reply:
Yeah, @sgober is this something you're comfortable adding with this PR?

Reply:
@holmesie up to you - I have a lot to accomplish with 1055 but am more than happy to add this now or in a future ticket

Reply:
@timwongj can you create a ticket with these details? Let me know if you want to do that together.

Reply:
I think the eslint stuff can be done in a separate ticket because it would potentially involve changing a lot of unrelated files as well

Reply:
here's the ticket: https://tracker.codev.mitre.org/browse/SARAALERT-1399

Reply:
thanks @timwongj!